WiinUPro/DevicePrefs.cs

```diff
@@ -26,6 +27,7 @@ public void Copy(DevicePrefs other)
             {
                 calibrationFiles.Add(calibration.Key, calibration.Value);
             }
+            extensionProfiles = other.extensionProfiles;

```

Comment:
Like mentioned before, copying instead of pointing at the existing object. I think this is the only change I feel like needs to be addressed this pull.